packages/elements/src/components/ino-label/ino-label.scss

```diff
@@ -1,7 +1,7 @@
-@use '../styles/mdc-customize';
+@use 'styles/mdc-customize';
+@use 'styles/colors';
```

Comment:
As we don't use colors inside this file, we don't need to import the colors file.